slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py

```diff
@@ -660,7 +670,7 @@ def setMRMLDefaults(self):
         self.scriptedEffect.setParameterDefault("ModelFilterLabel", "")
         self.scriptedEffect.setParameterDefault("SegmentationModel", "")
         self.scriptedEffect.setParameterDefault("AnnotationModel", "")
-        self.scriptedEffect.setParameterDefault("AnnotationModelFiltered", 1)
+        self.scriptedEffect.setParameterDefault("AnnotationModelFiltered", 0)
```

Comment:
what does this change combine with line 792 gives us

Reply:
by default we don't want enable it.. couple of review comments from https://github.com/NVIDIA/ai-assisted-annotation-client/issues/59#issuecomment-592296387